src/main.rs

```diff
@@ -0,0 +1,126 @@
+//! Actix Basic
+//!
+//! Tasks defined `rust_actix-basic.pdf`
+//!
+//! - Modeling endpoint behavior base on [this test api](https://jsonplaceholder.typicode.com/api/v1/users)
+//! - This server implementation based on [this basic example](https://github.com/actix/examples/blob/master/basics/src/main.rs)
+
+use simplelog::*;
+use std::{env, io};
+use validator::Validate;
+use::serde_json;
+
+#[macro_use]
+extern crate actix_web;
+use actix_files as fs;
+use actix_web::http::StatusCode;
+use actix_web::{error, middleware, web, App, HttpResponse, HttpServer, Result, Error};
+
+mod user;
+
+
+/// favicon handler
+#[get("/favicon.ico")]
+async fn favicon() -> Result<fs::NamedFile> {
+    Ok(fs::NamedFile::open("static/favicon.ico")?)
+}
+
+/// simple index handler
+#[get("/")]
+async fn welcome() -> Result<HttpResponse> {
+    Ok(HttpResponse::build(StatusCode::OK)
```

Comment:
Something you could replace this with is a healthcheck.  It's most typical for the endpoint to be called "health" and it's before the `/api/v1` portion of the URI.  That is, it's not versioned.  So, `http://localhost/health`.  Just return a 200 OK response with the string "ok" in the response body.